docs/source/tutorials/getting-started/transfer.rst

```diff
@@ -198,13 +202,18 @@ If you look at the implementation of the ``Transfer`` workflow, you will notice
   :start-after: -- DO_TRANSFER_BEGIN
   :end-before: -- DO_TRANSFER_END
 
-The first line converts the holding contract id (of type ``ContractId Holding.I``) to the
-``Transferable`` interface using ``coerceContractId``.
+The first line converts the holding contract id (of type
+:ref:`ContractId Holding.I <type-daml-finance-interface-holding-base-base-14854>`) to the
+:ref:`Transferable.I <type-daml-finance-interface-holding-transferable-transferable-24986>`
+interface using ``coerceContractId``.
```

Comment:
I missed that we have `coerceContractId` in the quick starter... argh!

Reply:
me too 😓 I think we can change this in the daml repo (but not our version, as we are on an old SDK version)

Reply:
we should be able to change it in both repos as I've already done part of this on the Daml-finance repo (there's only a few references left which I'll look into). That being said, this can be done as part of separate PR

Reply:
me too, we could have replaced a few `coerceContractId` by `toInterfaceContractId` and `fromInterfaceContrractId`, namley these:
`archive' self = archive (coerceContractId self : ContractId Account)`
`allocation = Pledge $ coerceContractId bobHoldingCid`
`allocation = Pledge $ toInterfaceContractId aliceHoldingCid`
For the transfer we would still need the `coerceContractId`:
`let transferableCid : ContractId Transferable.I = coerceContractId holdingCid`